### PR TITLE
DROOLS-6726 : Delete BRL column in model view return error

### PR DESCRIPTION
**Pull request by Rikkola**
**Thank you for submitting this pull request**

**JIRA**: 

[DROOLS-6726](https://issues.redhat.com/browse/DROOLS-6726)

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by Rikkola** [comment]
I'll remove the code smell soonish.

**Comment by jomarko** [comment]
Jenkins do fdb

**Comment by Rikkola** [comment]
@jomarko Ok, local build green without code smells. Ready to be reviewed.

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/table/model/synchronizers/impl/BRLConditionColumnSynchronizerTest.java [hunk 1]
```diff
@@ -577,6 +577,44 @@ public void testDelete() throws VetoException {
...
+        assertEquals(1,
+                     model.getConditions().size());
+
+        assertEquals(5,
+                     uiModel.getColumns().size());
+
+        modelSynchronizer.deleteColumn(columnV0);
+
+        assertEquals(4,
+                     model.getExpandedColumns().size());
```
Comment (jomarko) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/table/model/synchronizers/impl/BRLConditionColumnSynchronizerTest.java:610]:
```suggestion
                     model.getExpandedColumns().size());
        assertEquals("col1v1",
                     model.getExpandedColumns().get(3).getHeader());
```

**Review by jomarko — COMMENTED** [review]
@Rikkola thank you, one more suggested change to be sure the remained column is the expected one.

**Review by jomarko — APPROVED** [review]
Thank you @Rikkola

**Comment by Rikkola** [comment]
Jenkins run fdb
I guess I should give proof that this does not break GWT builds.

**Comment by jomarko** [comment]
@Rikkola I do not think FDB is under risk this time.
- We didn't change public API
- We didn't introduce new dependency

From my point of view this is ready for a merge

**Comment by yesamer** [comment]
Jenkins run fdb

**Comment by Rikkola** [comment]
Ok let's go then, two of us build the GWT bits already, should be fine.

@kiegroup/gatekeepers Can I get a merge please.

**Pull request merged**